packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -428,8 +428,11 @@ export class TdsDropdown {
   };
 
   private setValueAttribute = () => {
-    if (this.value?.toString() === '') this.value = null;
```

Comment:
Before we were doing `this.value = null;`, just need to make sure this is being done somewhere else even after this change.

Reply:
This is happening inside internalReset function right? @mJarsater I do not think we need to add it elsewhere. Am I wrong?